### PR TITLE
feature: load roles from omniauth

### PR DESCRIPTION
**Pull request by diegosteiner**

### CODE REVIEW

**Comment by cleverer** [comment]
Just to be sure: this doesn't affect the leaders, coaches etc access, right?

**Comment by diegosteiner** [comment]
> Just to be sure: this doesn't affect the leaders, coaches etc access, right?

@cleverer Natüürlich! (**checkt im sourcecode nach ob das wirklich so ist**) ... Die normalen Lagerleiter, AL und Coach-Logins sind nicht betroffen. Die Rolle user braucht es gar nicht.

**Review by cleverer — APPROVED** [review]
Users are also in Keycloak, go ahead

File: spec/models/user_spec.rb [hunk 1]
```diff
@@ -96,6 +96,15 @@
         expect(user.role_flags).to eq(0b0101)
       end
     end
+
+    it do
+      user.update(role_admin: true)
```
Comment (sihu) [inline spec/models/user_spec.rb:101]: ok, that means all existing roles in midata will be overwritten. this also means we no longer can give people access to things via orca

**Review by sihu — COMMENTED** [review]
LGTM 👍 nice job

File: spec/models/user_spec.rb (hunk 1 above)
Comment (cleverer) [reply spec/models/user_spec.rb:101]: jup, but via keycloak, which makes it a bit easier for support to handle.

**Pull request merged**